docs/core/diagnostics/index.md

```diff
@@ -49,6 +49,10 @@ The [dotnet-gcdump](dotnet-gcdump.md) tool is a way to collect GC (Garbage Colle
 
 .NET Core includes what is called the `EventPipe` through which diagnostics data is exposed. The [dotnet-trace](dotnet-trace.md) tool allows you to consume interesting profiling data from your app that can help in scenarios where you need to root cause apps running slow.
 
+### dotnet-stack
```

Comment:
There is another place you should try to add which is under `docs\fundamentals\diagnostics\toc.yml` which has links to each tools as a side panel that shows up in https://docs.microsoft.com/en-us/dotnet/core/diagnostics/